spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreOperations.java

```diff
@@ -57,7 +60,21 @@
 	<T> void deleteById(Object id, Class<T> entityClass);
 
 	/**
-	 * Delete an entity from Cloud Datastore.
+	 * Delete multiple IDs from Cloud Datastore. Deleting IDs that do not exist in Cloud Datastore
+	 * is OK.
+	 * @param ids the IDs to delete. If any of these is actually a
+	 * {@link com.google.cloud.datastore.Key}
+	 * then it will be used. Otherwise it will be attempted to be converted
+	 * to an integer or string value and it will be assumed to be a root key value with the
+	 * Kind determined by the entityClass.
+	 * @param entityClass the type of the
+	 * @param <T> ths entity type
+	 */
+	<T> void deleteAllById(Iterable<?> ids, Class<T> entityClass);
```

Comment:
Let's return count.

Reply:
We won't be aware of how many are actually deleted, because the client lib's delete function returns void and literally says that deleting non-existing keys is "OK". So, it will not be possible to give a count unless we check separately to see how many of those IDs exist beforehand.